.github/workflows/nightly-ami.yml

```diff
@@ -0,0 +1,96 @@
+name: Nightly AMI Build
```

Comment:
Putting the nightly build into GH actions puts us in a state where we do our release builds on AWS (+ all our other builds for all our other services), and only our nightly build in GH actions. I think we should move this to CodePipeline also, and use an event bridge rule to trigger the build nightly: https://docs.aws.amazon.com/codepipeline/latest/userguide/pipelines-trigger-source-schedule.html